lib/plugins/audit.js

```diff
@@ -91,6 +91,7 @@ function auditLogger(options) {
 
     function audit(req, res, route, err) {
         var latency = res.get('Response-Time');
+        var server = options.server;
```

Comment:
Cache the server up top when the audit function is initialized in its own variable.